feature_annotations.yaml

```diff
@@ -0,0 +1,27 @@
+# This code-annotations configuration file supports OEP-17, Feature Toggles.
+
+source_path: ./
+report_path: reports
+safelist_path: .annotation_safe_list.yml
+coverage_target: 50.0
+annotations:
+  documented_elsewhere:
+    - ".. documented_elsewhere_name:":
+  feature_toggle:
+    - ".. toggle_name:":
+    - ".. toggle_type:":
+        choices: [switch, rollout, group, ConfigurationModel, feature_flag]
+    - ".. toggle_default:":
+    - ".. toggle_description:":
+    - ".. toggle_category:":
+    - ".. toggle_use_cases:":
+        choices: [incremental_release, launch_date, monitored_rollout, graceful_degradation, beta_testing, vip, opt_out, open_edx]
+    - ".. toggle_creation_date:":
```

Comment:
Perhaps this is too nit-picky, but I feel like it would be easy to have incorrect data in this field. Say, for instance, you begin work on some code and annotate a flag using the current date, but the branch is long-lived and doesn't get merged for quite some time. I'm not sure how important accuracy is here. Since one of the purposes of annotating this data is so that it can be consumed by a report generator, what do you think about the tool determining this value? There are programmatic ways of determining when something was added to the codebase.